### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by bobturneruk**
Changes to readme and contributing to make them look nicer and be more easy to read and that. Also info on planned software architecture (which can totally change as we go along, just want some kind of plan). More detail on how to use the architecture to follow in #72 .

### CODE REVIEW

**Comment by bobturneruk** [comment]
Resolves #61 insofar as such a thing can ever be fully resolved!

**Comment by bobturneruk** [comment]
Resolves #22 sufficiently to close, but more work on the "How it works" sections would be helpful as part of a separate pull request.

File: contributing.md [hunk 1]
```diff
@@ -5,23 +5,35 @@ This document explains how, technically, to contribute to this project. A code o
...
+## Software Architecture
+
+`topostats` is currently arranged as a Python module in the `topostats/` folder. Other scripts in this repository are not currently considered core elements of `topostats` apart from `Plotting.py`.
+
+Currently the `topostats` module consists of:
+
+* `default_config.ini` The default config file.
+* `dnatracing.py`
+* `pygwytracing.py` The "main" routine.
+* `tracingfuncs.py`
```
Comment (MaxGamill-Sheffield) [inline contributing.md:24]: do dnatracing.py and tracingfuncs.py also need brief descriptors? Something like dnatracing - applies tracing functions to each molecules, and for tracingfuncs - skeletonises and generates backbone traces from masks (I think that's what they do at least)

**Review by MaxGamill-Sheffield — APPROVED** [review]
Just one comment about defining the other module constituents to be consistent

**Review by SylviaWhittle — APPROVED** [review]

File: contributing.md (hunk 1 above)
Comment (bobturneruk) [reply contributing.md]: Good idea. I think these will be subject to further revisions as well.

**Pull request merged**